src/core/api/public_api.ts

```diff
@@ -545,7 +553,12 @@ class Player extends EventEmitter<IPublicAPIEvent> {
       .subscribe((x : TextTrack[]) => this._priv_onNativeTextTracksNext(x));
 
     this._priv_playing$ = new ReplaySubject(1);
-    this._priv_speed$ = new BehaviorSubject(videoElement.playbackRate);
+    this._priv_speeds = {
+      speed$: new BehaviorSubject(videoElement.playbackRate),
+      trickModeSpeed$: new BehaviorSubject(videoElement.playbackRate),
+    };
+
+    this._priv_trickModeEnabled$ = new BehaviorSubject(false as boolean);
```

Comment:
Why casting here?
You can explicitly set the generic argument of Behavior subject if you encountered typing issue, it's better checked by TS than casting.